src/leo-server-app/src/routes/schedule.ts

```diff
@@ -313,6 +321,33 @@ router.get(
   }
 );
 
+router.get(
+  "/getScheduleBySatelliteAndTime",
+  async (req: GetScheduleBySatelliteAndTimeProp, res: any) => {
+    const {
+      satelliteId,
+      status = ScheduleStatus.FUTURE,
+      time,
+    } = req.query;
+
+    const convertedTime = new Date(time);
+
+    const filter = {
+      satelliteId: satelliteId,
+      status: status,
+      $and: [
+        {startDate: { '$lte': convertedTime }} ,
+        { endDate: { '$gte': convertedTime } },
+      ],
```

Comment:
May be wrong but shouldn't this be switched?

Reply:
it says if start date is less than equal to input time or end date greeater or eqaul to input time